src/core/Modal/Modal.tsx

```diff
@@ -0,0 +1,42 @@
+import {
+  Modal as MuiModal,
+  Button,
+  ModalProps as ModalBaseProps,
```

Comment:
We can have this is base.ts file

Reply:
@S-ayanide and I discussed this as we do not have any ModalProps which needed to be Omitted so we can have it inside the` componet.tsx` file itself.